style.css

```diff
@@ -24,27 +17,36 @@ body {
     font-size: 20px;
     color: #929292;
     font-style: normal;
-    margin: 0px 70px 35px;
 }
 
-.search-input {
-    text-transform: initial;
-    color: #929292; 
-    align-content: center;
+form {
```

Comment:
@ChrysaLynn 
This is a pretty basic flex setup. 

Note that you don't _have_ to set the `flex-direction: row;` property, as that's the default, but this comes out of habit for me. 😆 

Actually...the `align-items: center;` comes from previous habits, too.